net-mail/mu/mu-1.2.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools elisp-common
+
+DESCRIPTION="Set of tools to deal with Maildirs, in particular, searching and indexing"
+HOMEPAGE="http://www.djcbsoftware.nl/code/mu/"
+SRC_URI="https://github.com/djcb/mu/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="emacs zsh-completion"
+
+# net-mail/mailutils also installes /usr/bin/mu.  Block it until somebody
+# really wants both installed at the same time.
+DEPEND="
+	dev-libs/gmime:3.0
+	dev-libs/xapian
+	dev-libs/glib:2
+	emacs? ( >=virtual/emacs-24 )
+	zsh-completion? ( app-shells/gentoo-zsh-completions )
+	!net-mail/mailutils"
+RDEPEND="${DEPEND}"
+
+SITEFILE="70mu-gentoo.el"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	# Todo: Make a guile USE-flag as soon as >=guile-2 is avaiable
+	# Note: --disable-silent-rules is included in EAPI-5
+	econf --disable-guile \
+		--disable-gtk \
+		--disable-webkit \
+		$(use_enable emacs mu4e)
+}
+
+src_install () {
+	dobin mu/mu
+	dodoc AUTHORS HACKING NEWS NEWS.org TODO README README.org ChangeLog
+	if use emacs; then
+		elisp-install ${PN} mu4e/*.el mu4e/*.elc
+		elisp-site-file-install "${FILESDIR}/${SITEFILE}"
+		doinfo mu4e/mu4e.info
+	fi
+
+	if use zsh-completion; then
+		insinto /usr/share/zsh/site-functions
+		newins contrib/mu-completion.zsh _${PN}
+	fi
+
+	# TODO: Add guile man page when guile-2 is available.
+	doman man/mu-add.1 man/mu-bookmarks.5 man/mu-cfind.1 man/mu-easy.1 \
+		  man/mu-extract.1 man/mu-find.1 man/mu-help.1 man/mu-index.1 \
+		  man/mu-mkdir.1 man/mu-query.7 man/mu-remove.1 man/mu-script.1 \
+		  man/mu-server.1 man/mu-verify.1 man/mu-view.1 man/mu.1
```

Comment:
This is silly. Use Bash expansion to shorten this up:

`doman man/mu*`

If any man pages need to be excluded, just remove them from the image on the next line. However, it looks like they're all needed.